test/kt/esimd_radix_sort.cpp

```diff
@@ -434,7 +434,14 @@ main()
                 test_general_cases<TEST_DATA_TYPE, Ascending, RadixBits>(q, size, kernel_parameters);
                 test_general_cases<TEST_DATA_TYPE, Descending, RadixBits>(q, size, kernel_parameters);
             }
-            test_small_sizes<TEST_DATA_TYPE, Ascending, RadixBits>(q, kernel_parameters);
+            // TODO: enable that case once compilation issue is fixed:
+            // The compiler produces "Exception: internal compiler error" with some configurations,
+            // e.g. with 256 DataPerWorkItem and 64 WorkGroupSize.
+            // This issue is revealed by a change when slm_init in onesweep impl is supplied with an argument which depends on input data;
+            // it does not show up when the argument is a hardcoded number.
+            // Workaround: use AOT compilation or pass -fsycl-device-code-split=per_kernel flag during JIT compilation.
```

Comment:
May we use this option `-fsycl-device-code-split=per_kernel` and uncomment this test?

Reply:
I restored the test case.